misk-slack/src/main/kotlin/misk/slack/webapi/SlackClient.kt

```diff
@@ -0,0 +1,15 @@
+package misk.slack.webapi
+
+import misk.slack.webapi.helpers.PostMessage
+import misk.slack.webapi.helpers.PostMessageResponse
+
+interface SlackClient {
+  fun postMessage(
+    request: PostMessage,
```

Comment:
I'd lean towards this being renamed to `PostMessageRequest` for symmetrical naming between the request / response classes.